### ISSUE TITLE
Cannot build sample application with python toolchain

### ISSUE THREAD

**Issue by holler2k21** [qooxdoo/qooxdoo#10253]
**Describe the bug**
Cannot build sample application with python toolchain in qooxdoo release 6.0.2.  
Build with python toolchain fails with the following error:
`Unknown operator u'let' (pos (375, 9))`

**To Reproduce**
- unzip last qooxdoo-release_6_0_2.zip to directory
- open shell in directory application\tutorial
- run generate.py
- output
```
----------------------------------------------------------------------------
    Executing: source-hybrid::source-script
----------------------------------------------------------------------------
    - Populating the cache, this may take some time
>>> Scanning libraries
>>> Collecting classes  -<class 'ecmascript.frontend.SyntaxException.SyntaxException'> :
Unknown operator u'let' (pos (375, 9))
File: qooxdoo-release_6_0_2\framework\source/class\qx\data\Array.js
```

**Expected behavior**
Build should succeed.

**Environment**
 - Windows 10
 - Python 2.7

**Comment by hkollmann** [comment]
Could you fix it and create a PR?

Thanks!

**Comment by hkollmann** [comment]
Is fixed in release 6.0.3
